Makefile

```diff
@@ -529,5 +529,6 @@ clean:
 
 distclean: clean
 	rm -rf node_modules
+	rm -rf build
```

Comment:
Nope, because `$(BUILD_DIRECTORY)` is `dist` (which already gets deleted by the `clean` rule).